src/functions/function-tree-view/functionsTreeItem.ts

```diff
@@ -126,7 +126,7 @@ export class FunctionNodeImpl implements FunctionNode {
       this.contextValue === FunctionContextType.NOTCONNECTEDLOCALFUNCTIONS
     ) {
       return format(
-        `Name: ${this.CONTEXT_DATA[this.contextValue].tooltip} RunTime: ${this.runtime} Context: ${this.contextPath.fsPath}`,
+        `Name: ${this.CONTEXT_DATA[this.contextValue].tooltip}\nRuntime: ${this.runtime}\nContext: ${this.contextPath.fsPath}`,
```

Comment:
@sudhirverma is runtime the right word, or Language ? As in the UI we ask for `Select Language` ?

Reply:
In func.yaml it displays `runtime: node` so I show runtime in tooltip.

<img width="677" alt="Screenshot 2022-04-11 at 11 10 32 AM" src="https://user-images.githubusercontent.com/9924513/162671930-22bd210d-d88d-41de-80d5-6630e707b3d4.png">

Reply:
and in func list, it shows runtime.

```
func list -o json
[{"name":"test-demo-1","namespace":"default","runtime":"node","url":"http://test-demo-1.default.example.com","ready":"True"}]
```

Reply:
okay then it works.